.prettierrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  singleQuote: true,
+  trailingComma: 'es5',
+  semi: false,
```

Comment:
do not use semicolon at the end of lines because cypress recommends don't use them